packages/switchToggle/src/Default/Default.types.ts

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import * as bs from 'react-bootstrap';
+
+export interface SwitchToggleContainerProps {
+  id: string;
+  toggle: boolean;
+  label: string;
+  type: string;
+}
+
+export interface SwitchToggleDefaultCommons {
+  theme?: 'Default' | 'Dark';
+  size?: 'Sm' | 'Md' | 'Lg';
+  prefix?: string;
+  children: React.ReactNode;
+  switchtoggle: SwitchToggleContainerProps;
```

Comment:
When using the `intersection`, all types of the `bs.FormCheckProps` will also be added to the `SwitchToggleDefaultCommons` automatically, which means you don't need the `switchtoggle: SwitchToggleContainerProps;`.